src/configs/recommended.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @license
+ * Copyright 2016 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const rules = {
+    "align": [true,
+        "parameters",
+        "statements",
+    ],
+    "class-name": true,
+    "comment-format": [true,
+        "check-space",
+    ],
+    "curly": true,
+    "eofline": true,
+    "forin": true,
+    "indent": [true, "spaces"],
+    "interface-name": true,
```

Comment:
Actually, thinking this is the wrong setting for the rule overall. Having an `I` prefix is like naming variable `length_number` and `name_string` and is redundant info